frontend/src/app/plugins/local_storage.cljs

```diff
@@ -0,0 +1,71 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.plugins.local-storage
+  (:require
+   [app.common.data.macros :as dm]
+   [app.common.exceptions :as ex]
+   [app.plugins.register :as r]
+   [app.plugins.utils :as u]
+   [app.util.globals :as g]
+   [app.util.object :as obj]
+   [cuerdas.core :as str]))
+
+(defonce ^:private local-storage
+  (ex/ignoring (unchecked-get g/global "localStorage")))
+
+(defn prefix-key
+  [plugin-id key]
+  (dm/str plugin-id "/" key))
```

Comment:
I think you can follow a similar scheme as we already using on penpot. I suggest changing this to:

`(dm/str "penpot-plugins:" plugin-id "/" key)`

Reply:
OK